core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1250,21 +1250,23 @@ private void updateMediaSession(final PlayerStatus playerStatus) {
 
         UiModeManager uiModeManager = (UiModeManager) getApplicationContext()
                 .getSystemService(Context.UI_MODE_SERVICE);
-        if (uiModeManager.getCurrentModeType() == Configuration.UI_MODE_TYPE_CAR) {
+        // Hack to work around this bug https://github.com/AntennaPod/AntennaPod/issues/5481#issuecomment-999811393
```

Comment:
Could you instead link the Google bug, please? Then one doesn't have to click the link to see that it works around an Android bug, not an AntennaPod bug.